src/config-v2.json

```diff
@@ -112,6 +112,7 @@
                 "pacientelor",
                 "bolnavi",
                 "bolnave",
+                "îmbolnăvi",
```

Comment:
Can we remove this?

Reply:
why? it matches just part of the word if I have îmbolnăviri in an article it will highlight îm[bolnavi] instead of the full word